Formula/mtoc.rb

```diff
@@ -24,6 +25,11 @@ class Mtoc < Formula
     sha256 "0d20ee119368e30913936dfee51055a1055b96dde835f277099cb7bcd4a34daf"
   end
 
+  patch do
+    url "https://raw.githubusercontent.com/acidanthera/ocbuild/8107959e2a54122516d2f1fe5797e6f906a4fddc/patches/mtoc-add-missing-efi-subsystems.patch"
+    sha256 "463764cfd2027cbaba7001e7d5d28e06917aedb866a1c2da664a6f3dcff21636"
+  end
```

Comment:
Can you add a test case for this?

Reply:
I am afraid that will require cloning and compiling a EDK II package, this will take a lot of time, and is impractical.